templates/sonarqube-values.yaml

```diff
@@ -0,0 +1,47 @@
+image:
+  tag: 7.6-community
+
+service:
+  type: ClusterIP
+
+ingress:
+  enabled: true
+  hosts:
+    - name: [HOSTNAME]
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    kubernetes.io/tls-acme: "true"
+    certmanager.k8s.io/cluster-issuer: [CLUSTER_ISSUER]
+    ingress.kubernetes.io/secure-backends: "true"
+  tls:
+    - secretName: sonar-tls-[CLUSTER_ISSUER]
+      hosts:
+        - [HOSTNAME]
+    
+metadata:
+  name: basic-ingress
+
+persistence:
+  size: 10Gi
+  accessMode: ReadWriteOnce
+
+postgresql:
+  persistence:
+    size: 10Gi
+
+
+sonarProperties:
+  sonar.forceAuthentication: true
+
+# https://docs.sonarqube.org/display/PLUG/Plugin+Version+Matrix
```

Comment:
Avoid using comments for disabling code.  Remove code or document why it's disabled.